carp.studies.core/src/commonMain/kotlin/dk/cachet/carp/studies/application/RecruitmentServiceHost.kt

```diff
@@ -204,4 +277,17 @@ class RecruitmentServiceHost(
     private suspend fun getRecruitmentOrThrow( studyId: UUID ): Recruitment =
         participantRepository.getRecruitment( studyId )
             ?: throw IllegalArgumentException( "Study with ID \"$studyId\" not found." )
+
+    private suspend fun getRecruitmentAndGroupByGroupIdOrThrow(
```

Comment:
I think you can safely drop "ByGroupId" as that is already documented by the parameter name.